src/workloads/encrypted/qe-range-age-100-0.yml

```diff
@@ -0,0 +1,1036 @@
+SchemaVersion: 2018-07-01
+Owner: "@10gen/server-security"
+Description: |
+  QE Range Release Criteria Experiment
+Keywords:
+- Queryable Encryption
+
+GlobalDefaults:
+  MyDatabase: &encrypted_db genny_release_criteria
+  MyCollection: &encrypted_coll my_coll
+  NopAlias: &nop {Nop: true}
+  
+  StateTransitions: &transitions
+  - To: Query
+    Weight: 1
+    SleepBefore: {^TimeSpec: {value: 0, unit: seconds}}
+  - To: Update
+    Weight: 0
+    SleepBefore: {^TimeSpec: {value: 0, unit: seconds}}
+  
+
+
+Encryption:
+  
+  UseCryptSharedLib: true
+  CryptSharedLibPath: /data/workdir/mongocrypt/lib/mongo_crypt_v1.so
+  
+  EncryptedCollections:
+  - Database: *encrypted_db
+    Collection: *encrypted_coll
+    EncryptionType: queryable
+
+    QueryableEncryptedFields:
+      tm_retail_tx: { type: "long", queries: [{queryType: "range", min: 0, max: 1000000000000, contention: 4, sparsity: 2, trimFactor: 8}] }
+      age_hospitals: { type: "int",  queries: [{queryType: "range", min: 0, max: 55000, contention: 8, sparsity: 2, trimFactor: 6}] }
+      bnk_bal: { type: "decimal", queries: [{queryType: "range", min: 0, max: 1000000, precision: 2, contention: 8, sparsity: 2, trimFactor: 6}] }
+
+
+Clients:
+  EncryptedPool:
+    QueryOptions:
+      maxPoolSize: 400
+  
+    EncryptionOptions:
+      KeyVaultDatabase: "keyvault"
+      KeyVaultCollection: "datakeys"
+      EncryptedCollections:
+      - genny_release_criteria.my_coll
+  
+
+Actors:
+
+  
+  
+  - Name: InsertActor_Thread0
+    Type: CrudActor
+    Threads: 1
+    Database: *encrypted_db
+    ClientName: EncryptedPool
+    Phases:
+    - *nop
+    - Repeat: 6250
+      Collection: *encrypted_coll
+      MetricsName: "load"
+      Operations:
+      - OperationName: insertOne
+        OperationMetricsName: inserts
+        OperationCommand:
+          Document:
+            tm_retail_tx: {^ConvertToInt: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/data/rc_timestamps.txt", "sequential": true, "startFromLine": 0}}}}
```

Comment:
Can this path be made relative? It may not be possible but I don't see how I would run this workload locally without having to artificially create the DSI directory layout which would be irritating.

Although the ```CryptSharedLibPath``` also makes this difficult.

Reply:
The path already is relative; this is just how the directory structure is set up on Evergreen. Unfortunately I don't think there's a way to make a workload which has paths that work for both local and Evergreen, which is why I write to the separate `workloads/local` and `workloads/evergreen` paths in my experiment generation script. This is an existing problem though; see https://github.com/mongodb/genny/blob/1ce2f0de326b66764d5bdb66349178c386a5a168/src/workloads/docs/ChooseFromDataset.yml#L8-L9

Reply:
I've raised DEVPROD-7205 for this. It's a bit brittle and annoying.